### PR TITLE
Refactor test 

### PR DESCRIPTION
**Pull request by ikerreyes**
This PR mainly affects table related test suites.

The goal is to try to reduce common code and nest tests into meaningful categories.

Create a new method to get ouia components by type and rename the one for IDs
Add new components to the global file
Set some common table related utilities

### CODE REVIEW

**Review by Fewwy — APPROVED** [review]
Looks good to me!

File: src/Components/ClusterRules/ClusterRules.spec.ct.js [hunk 1]
```diff
@@ -191,155 +175,182 @@ describe('cluster rules table', () => {
...
+      });
+    });
+    describe('combined filters', () => {
+      filterCombos.forEach((filters) => {
+        it(`${Object.keys(filters)}`, () => {
+          const sortedDescriptions = _.map(
+            filterData(data, filters),
+            'description'
+          ).sort();
+          // debugger;
```
Comment (gkarat) [inline src/Components/ClusterRules/ClusterRules.spec.ct.js:321]:
```suggestion
```

**Review by gkarat — APPROVED** [review]
superficially, looks good to me 👍🏼 I didn't have enough time to go through each test; but since they pass and syntactically is ok, then I am fine with it :D

**Pull request merged**